core/src/test/java/org/apache/spark/JavaAPISuite.java

```diff
@@ -63,6 +63,7 @@
 // The test suite itself is Serializable so that anonymous Function implementations can be
 // serialized, as an alternative to converting these anonymous classes to static inner classes;
 // see http://stackoverflow.com/questions/758570/.
+@SuppressWarnings("serial")
```

Comment:
I would not bother adding these. This is a silly javac lint warning and should be disabled at the compiler level, IMHO.